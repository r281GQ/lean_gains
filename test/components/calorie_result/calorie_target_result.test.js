import React from 'react';
import { shallow } from 'enzyme';
import CalorieTargetResult from './../../../src/components/calorie_result/calorie_target_result';

describe('CalorieTargetResult test', () => {
  it('should render', () => {
    const component = shallow(<CalorieTargetResult />);
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
