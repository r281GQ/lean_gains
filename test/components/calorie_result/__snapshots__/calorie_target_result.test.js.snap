// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalorieTargetResult test should render 1`] = `null`;
